### ISSUE TITLE
Docs build fails with firebase JS file access error

### ISSUE THREAD

**Issue by teor2345** [ZcashFoundation/zebra#514]
When I squashed and merged #502 to main, I got the following error
```
> storage-engine@3.0.7 postinstall /usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine
> node ./compatibility/detect.js

fs.js:114
    throw err;
    ^

Error: EACCES: permission denied, open '/usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/index.js'
    at Object.openSync (fs.js:443:3)
    at Object.writeFileSync (fs.js:1194:35)
    at /usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/detect.js:35:15
    at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/detect.js:45:3)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
```
https://github.com/ZcashFoundation/zebra/runs/793400595

**Comment by hdevalence** [comment]
I think this may have been fixed in #518

**Comment by dconnolly** [comment]
Resolved by #518
